templates/taxi/car_list.html

```diff
@@ -1,16 +1,49 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>Car list</h1>
+  <h1>Car list
+  <a class="btn btn-primary" href="{% url 'taxi:car-create' %}">Create</a>
+  </h1>
   {% if car_list %}
-    <ul>
+    <table class="table">
+    <tr>
+          <td>
+            ID
+          </td>
+        <td>
+            Model
+          </td>
+        <td>
+            Manufacturer
+          </td>
+        <td>
+            Delete
+          </td>
+        <td>
+            Update
+          </td>
```

Comment:
>
2. On the car list page add button that leads to the car creation page.
**On the car detail page add buttons that lead to the car update page and car deletion page.**

Reply:
They are available at car list (button create) and in car detail page (two buttons update and delete) 
 <h1>Car list
  <a class="btn btn-primary" href="{% url 'taxi:car-create' %}">Create</a>
  </h1>
  
  <div>
      <a  class="btn btn-primary" href="{% url 'taxi:car-update' pk=car.id%}">Update</a>
      <a class="btn btn-primary" href="{% url 'taxi:car-delete' pk=car.id%}">Delete</a>
  </div>